src/sitegen/layout.cljs

```diff
@@ -37,7 +37,7 @@
     [:title title]
     [:meta {:name "description" :content description}]
 
-    [:link {:rel "stylesheet" :href "http://fonts.googleapis.com/css?family=Open+Sans:400,300,600,700"}]
+    [:link {:rel "stylesheet" :href "https://fonts.googleapis.com/css?family=Open+Sans:400,300,600,700"}]
```

Comment:
I believe if you just use: "//fonts.googleapis.com/css?family=Open+Sans:400,300,600,700" the protocol will be inferred. 

CDN below could utilize that as well.